CHANGELOG.md

```diff
@@ -18,6 +18,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - We changed some internal structures, but you shouldn't be too affected by them as these are internal changes.
 
+### Pool options types declaration merge
+
+`FixedThreadPoolOptions` and `DynamicThreadPoolOptions` type declarations have been merged to `PoolOptions<WorkerType>`
```

Comment:
`WorkerType` is a bit of a duplicate here as it is the generic type and cannot be anything different here.
Also the generic type in the code when using it provides `Worker` in e.g. tooltips.
Therefore we should stick to one wording and don't provide multiple kinds of the same thing.

```suggestion
`FixedThreadPoolOptions` and `DynamicThreadPoolOptions` type declarations have been merged to `PoolOptions<Worker>`.
```